externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -75,7 +78,9 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
   override def closeSession(sessionHandle: SessionHandle): Unit = {
     _latestLogoutTime = System.currentTimeMillis()
     super.closeSession(sessionHandle)
-    operationManager.removeSparkSession(sessionHandle)
+    val session = operationManager.removeSparkSession(sessionHandle)
+    info(s"${session.sparkContext.sparkUser}'s session with $sessionHandle is closed," +
```

Comment:
In a Spark SQL egine , `session.sparkContext.sparkUser` is static, so looks unnecessary here

Reply:
ACK